matRad_dijSampler.m

```diff
@@ -0,0 +1,63 @@
+function [cst,dij] = matRad_dijSampler(cst,ct,dij, margin_size,sparCity)
+% marginsize given in pixel thickness
+% fraction of voxels to be sampled [ Target OAR]
+
```

Comment:
complete function header is missing. also I think this script could be moved to the tools folder